docker/intel-dgpu-driver.Dockerfile

```diff
@@ -37,10 +38,14 @@ RUN git clone -b ${I915_RELEASE} --single-branch https://github.com/intel-gpu/in
     && cp defconfigs/drm .config \
     && make olddefconfig && make -j $(nproc) && make modules_install
 
+# Create /build/modules directory with all *.ko and *.ko.xz files
+RUN mkdir -p /build/modules && find /lib/modules/${KERNEL_FULL_VERSION}/ -name "*.ko" -o -name "*.ko.xz" -o -name "modules.order" -o -name "modules.builtin" -type f | xargs -I {} install -D {} /build/modules/
```

Comment:
Please use the continuation character \\ to make the line short and more readable. Please refer to line 45 to line 47.

Reply:
Done